Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -155,6 +164,7 @@ public final class PackageInfoMapper: PackageInfoMapping {
     // Predefined source directories, in order of preference.
     // https://github.com/apple/swift-package-manager/blob/751f0b2a00276be2c21c074f4b21d952eaabb93b/Sources/PackageLoading/PackageBuilder.swift#L488
     fileprivate static let predefinedSourceDirectories = ["Sources", "Source", "src", "srcs"]
+    fileprivate static let predefinedTestsDirectories = ["Tests", "Sources", "Source", "src", "srcs"]
```

Comment:
Shouldn't this be just `Tests`?

Reply:
This is what SPM have as default directories for test targets. I copied it from the link above

```swift
// Predefined test directories, in order of preference.
public static let predefinedTestDirectories = ["Tests", "Sources", "Source", "src", "srcs"]
```